caim_base/static/js/common.js

```diff
@@ -46,6 +46,29 @@ $('[data-timestamp]').each(function (idx, el) {
     el.text(fmt.format(dte));
 });
 
+
+$('.reply').click(function(button){
+    let buttonID = button.target.id;
+
+        // Reply form element
+    var replyForm = `
+    <form method="post" id="reply-form${buttonID}" class="reply-form" action="/comments/${buttonID}/reply">
+        {% csrf_token %}
+        <input type="text" name="comment_id" value="${buttonID}" hidden>
```

Comment:
this is not correct is it? Should be type="hidden" ?

Reply:
This does work because bootstrap hides things with the hidden attribute, however its more correct to use as this is what its intended for.